CHANGELOG.md

```diff
@@ -5,10 +5,12 @@ All notable changes to this project will be documented in this file.
 
 ### Added
 
--   #1529 : Add Python support for list method `append()`
+-   #1689 : Add Python support for list method `append()`
+-   #1692 : Add Python support for list method `insert()`
```

Comment:
Is this branch based off `List_Support/insert_method`?

Reply:
yes it is, `List_Support/insert_method` should be merged before this one